### DISCUSSION TITLE
Logical Namespace in Redis

### DISCUSSION THREAD

**Discussion by raghuchowdary1994** [redis/lettuce#2241] [Q&A]
Does Lettuce provide logical namespacing features to create multiple logical caches in Redis?

**Comment by raghuchowdary1994** [comment]
Similar to Jedis implementation here 
https://github.com/TrigonicSolutions/jedis-namespace

**Comment by mp911de** [comment]
No, Lettuce is a Redis driver that lets you connect to a Redis server without an opinion on how you intend to use Redis.
